Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md

```diff
@@ -31,23 +31,27 @@ An active Cloud Application Manager account.
 
 ### Instances page
 
-The left side menu of Applications site shows you at a glance a view of all instances you have access to through your configured providers.
+The Instances left side menu option of the Applications site allows you to see at a glance a view of all instances you have access to through your configured providers in your current scope.
 
 ![All instances page](../../images/cloud-application-manager/instances/all-instances.png)
 
-There are three different submenus under the Instances menu:
+There are three different submenu options under Instances:
 
 * **All**: shows all type of instances, either registered or unregistered.
 
 * **Registered**: shows only registered instances, which are the ones deployed through or imported into Cloud Application Manager. In these instances, you can do lifecycle management by executing lifecycle events or using the lifecycle editor to change variables, event scripts or box versions of the instance. You can also select a specific state among the available ones (Online, Unavailable, Processing, Shutdown, Terminated) to show only the instance in the selected state.
 
 * **Unregistered**: shows only unregistered instances, which are the ones accessible into all the defined providers that were not deployed through Cloud Application Manager. They are discovered in the synchronization event of a provider and are classified by Type and Subtype.
-  * Type: Any of Compute, Network, Database, Storage or Other
+  * Type: One of Compute, Network, Database, Storage or Other
   * Subtype: this is the instance class, specific to each provider type. For example, for AWS type providers we can see VPC or Application Load Balancers (for Network type instances), and S3 or Elastic Block Storages (for Storage type instances)
 
   You can also select a specific state among the available ones (Active, Inactive) to show only the instance in the selected state.
 
-A Compute type unregistered instance can be registered (imported) into Cloud Application Manager, to enable lifecycle management on it (an icon button allowing it will appear at the end of its row), so it will be shown from them on as a registered instance in the corresponding views. The Unregistered Instances tab in the provider details page remains unchanged showing only compute registerable instances. If you want to bulk register (import) several instances from the same provider, use this feature from there instead, where bulk register is also available as a bulk action.
+The Instances page displays on the top of the page a new button, a search field  and a filters and instance view buttons. The search field allows you to find any instance looking for its name, instance-id, service-id, public or private IP address, support_id or hostname fields, or also by owner or last user who acted on the instance.
```

Comment:
Apparently, there are many more fields an instance can be searched by, so I suggest to enumerate only the most important ones.

Reply:
Ok, done, thanks!

Reply:
You're welcome!